### DISCUSSION TITLE
Im at a loss on the point...

### DISCUSSION THREAD

**Discussion by spiritedusual** [nextcloud/all-in-one#4081] [Questions] [answered]
I don't post and I am not a programmer or anything like that. I had almost 0 experience in linux before this. It took three days to get nextcloud up and running. By the end I learned a lot. However, I am kind of wondering what the point of the free version is. After I finally was up and running I had crazy paths like files/admin/files/admin/files/admin/"user1"/files/admin/photos/. Also, I went from about 100gb of data to almost 300gb... like wtf? After days of working to understand how to install and just get it up and running it turns out it's just simply not good for my usecase because my usecase is too simple? I understand that for large companies you might want to have duplicates of every file for each user and isolate each users files for privacy etc... BUT... like there is a free version... that like would be perfect if it just like kept the files on a server accessible to like a family of 4???? Why wouldn't you just have options to keep it simple? Nextcloud wont run when new files are added to the server from the host location? WTF??? I have to run OCC manually or create script WTF??? Just update the damn folder when it detects changes! I have several native windows programs that do this. Why can't nextcloud do this? Like think syncthing with all the other stuff added on... and you have enterprise solutions for ya know enterprise... 

Is it like a linux thing I don't understnad linux people maybe??? What am I missing here?

**Comment by szaimen** [comment, marked as answer]
Hi, this kind of discussion is not suitable for GitHub. But feel free to post it at https://help.nextcloud.com
